### PR TITLE
feat(plugins): Add the `isWeekend` property

### PR DESCRIPTION
**Pull request by bugtype**
### Summary

- reopen https://github.com/veccu/react-calendar/pull/38

### CODE REVIEW

File: examples/basic/src/pages/index.tsx [hunk 1]
```diff
@@ -126,23 +126,51 @@ export default function BasicExample() {
...
+                    key,
+                    date,
+                    isCurrentDate,
+                    isCurrentMonth,
+                    isWeekend,
+                  } = day
+
+                  // FIXME: hard coding
+
+                  if (isWeekend) {
```
Comment (JaeYeopHan) [inline examples/basic/src/pages/index.tsx:139]: sadly, `examples/basic` project need to update dependency to next version, **after publishing new version** ... 😢

Comment (bugtype) [reply examples/basic/src/pages/index.tsx]: Okay~! I was thinking about checking `undefined`, but I'll update it to another issue later!
I reflected it.

**Review by JaeYeopHan — APPROVED** [review]
i think `isWeekend: true` test case will be needed! :)

thanks!

**Comment by JaeYeopHan** [comment]
FYI) change pr title :)

**Comment by JaeYeopHan** [comment]
@all-contributors please add @bugtype for code

**Comment by JaeYeopHan** [comment]
published in https://github.com/veccu/react-calendar/releases/tag/v2.3.0

**Pull request merged**